examples/getting_started/pendulum.py

```diff
@@ -67,12 +70,24 @@ def prepare_ocp(biorbd_model_path, final_time, number_shooting_points, nb_thread
 if __name__ == "__main__":
     ocp = prepare_ocp(biorbd_model_path="pendulum.bioMod", final_time=3, number_shooting_points=100, nb_threads=4)
 
+    # --- Single shooting --- #
+    X = InitialConditions([0, 0, 0, 0])
+    U = InitialConditions([-1, 1])
+    sol_simulate = Simulate.from_controls_and_initial_states(ocp, X, U)
+    result = ShowResult(ocp, sol_simulate)
+    result.graphs()
+
```

Comment:
Pendulum has gotten highly clustered because of all the random features added. For a new comer, it is not obvious that the sole example of Simulate would be dug into this particular file.
I suggest a specific file for each of the features in pendulum, limiting it to the actual pendulum thing!

Reply:
I agree, it should be in this Pr or in new PR ?

Reply:
What you have done was good (creating a new file for this)